### ISSUE TITLE
Page refresh causes from resubmission in user management in admin panel

### ISSUE THREAD

**Issue by artplan1** [spree/spree#8589]
## Context

Not sure if it's worth fixing, because it's too small

I created/updated users and somehow saw glitch with data in forms (can't reproduce), but when I tried to refresh page - it sends form again

## Expected Behavior

Send `GET` on refresh

## Actual Behavior

![image](https://user-images.githubusercontent.com/9060346/35968068-d584c9c6-0cd3-11e8-90a8-77e218cf8d77.png)

Started PUT "/admin/users/7" for 127.0.0.1 at 2018-02-08 13:17:19 +0300

## Possible Fix

Change:

```ruby
if @user.save
  flash.now[:success] = flash_message_for(@user, :successfully_created)
  render :edit
```

to:

```ruby
if @user.save
  flash[:success] = flash_message_for(@user, :successfully_created)
  return redirect_to edit_admin_user_path(@user)
```

## Steps to Reproduce

1. Go to user management in admin panel
2. Create or update user
3. Refresh page

## Your Environment

* Version used: spree edge

**Comment by mohsen-alizadeh** [comment]
After the user is created in Controller, it tries to render user edit page instead of redirecting to edit page.

I have fixed it locally, I'm trying to prepare some tests and reproduce script and create a merge request soon.

Please assign this issue to me

**Comment by mohsen-alizadeh** [comment]
Actually it should be redirect to edit user page instead of rendering edit user page

**Comment by bbonislawski** [comment]
Fixed with #8603
